src/uniprotkb/components/entry/tabs/genomic-coordinates/Coordinates.tsx

```diff
@@ -0,0 +1,178 @@
+import { InfoList, LongNumber, SpinnerIcon } from 'franklin-sites';
+
+import ExternalLink from '../../../../../shared/components/ExternalLink';
+import LazyComponent from '../../../../../shared/components/LazyComponent';
+import DatatableWithToggle from '../../../../../shared/components/views/DatatableWithToggle';
+import GenomicLoc, { getEnsemblLink } from './GenomicLoc';
+import Overlapping from './Overlapping';
+
+import useDatabaseInfoMaps from '../../../../../shared/hooks/useDatabaseInfoMaps';
+
+import { processUrlTemplate } from '../../../protein-data-views/XRefView';
+
+import { GenomicCoordinate } from './types';
+
+type CoordinatesProps = {
+  coordinates: GenomicCoordinate;
+  index: number;
+  taxID?: number;
+  currentEntry: string;
+};
+
+const Coordinates = ({
+  coordinates,
+  index,
+  taxID,
+  currentEntry,
+}: CoordinatesProps) => {
+  const { genomicLocation: gl } = coordinates;
+
+  const ensemblInfo = useDatabaseInfoMaps()?.databaseToDatabaseInfo.Ensembl;
+
+  if (!taxID || !ensemblInfo?.uriLink) {
```

Comment:
Playing devils advocate here - if taxID is null then `<h3>Genomic location {index + 1}</h3>` will present a gap eg

```
<h3>Genomic location 1</h3>
// no taxID => null
<h3>Genomic location 3</h3>
<h3>Genomic location 4</h3>
```

Would we want to first filter in the parent of this component before mapping over the elements to keep a non-gapped sequence?